itests/org.openhab.binding.dmx.tests/itest.bndrun

```diff
@@ -0,0 +1,39 @@
+-include: ../itest-common.bndrun
+
+Bundle-SymbolicName: ${project.artifactId}
+Fragment-Host: org.openhab.binding.dmx
+
+-runrequires: bnd.identity;id='org.openhab.binding.dmx.tests'
+
+#
+# done
+#
+-runbundles: \
+	com.google.gson;version='[2.7.0,2.7.1)',\
+	javax.measure.unit-api;version='[1.0.0,1.0.1)',\
+	org.apache.commons.io;version='[2.2.0,2.2.1)',\
+	org.apache.commons.lang;version='[2.6.0,2.6.1)',\
+	org.apache.felix.configadmin;version='[1.9.8,1.9.9)',\
+	org.apache.felix.scr;version='[2.1.10,2.1.11)',\
+	org.eclipse.equinox.event;version='[1.4.300,1.4.301)',\
+	org.osgi.service.event;version='[1.4.0,1.4.1)',\
+	tec.uom.lib.uom-lib-common;version='[1.0.2,1.0.3)',\
+	tec.uom.se;version='[1.0.8,1.0.9)',\
+	osgi.enroute.hamcrest.wrapper;version='[1.3.0,1.3.1)',\
+	osgi.enroute.junit.wrapper;version='[4.12.0,4.12.1)',\
+	org.openhab.core;version='[2.5.0,2.5.1)',\
+	org.openhab.core.config.core;version='[2.5.0,2.5.1)',\
+	org.openhab.core.storage.json;version='[2.5.0,2.5.1)',\
+	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
+	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
+	net.bytebuddy.byte-buddy;version='[1.7.9,1.7.10)',\
+	net.bytebuddy.byte-buddy-agent;version='[1.7.9,1.7.10)',\
+	org.apache.commons.collections;version='[3.2.1,3.2.2)',\
+	org.mockito.mockito-core;version='[2.13.0,2.13.1)',\
```

Comment:
This file will need to be regenerated because of the Mockito upgrade (https://github.com/openhab/openhab-core/pull/647).